src/cr/cube/matrix/subtotals.py

```diff
@@ -73,7 +68,7 @@ def _intersections(self):
                 for row_subtotal in self._row_subtotals
                 for column_subtotal in self._column_subtotals
             ],
-            dtype=self._dtype,
+            dtype=np.float64,
```

Comment:
do we still need an explicit `dtype` parameter here or can we just let the defaults take over?

Reply:
I don't think it'd error, but if empty it'd convert to a non-float, right? Is there value in ensuring that it'd be the same type as the other blocks?